server/controler/db.go

```diff
@@ -57,3 +57,13 @@ func (dbStore *DbStore) IsUserPresent(username string) (bool, error) {
 func (dbStore *DbStore) CreateUser(newUser *model.User) error {
 	return dbStore.db.Create(newUser).Error
 }
+
+func (dbStore *DbStore) EmailAlreadyInUse(email string) (bool, error) {
+	var users []model.User
+	result := dbStore.db.Where("email <> ?", email).Find(&users)
+	if result.RowsAffected > 0 {
+		return true, nil
+	} else {
+		return false, nil
+	}
```

Comment:
I think we should check if the SQL driver does not return an error

Reply:
May I add this when DB script is fixed?

Reply:
I cannot put my finger what's the reason to postpone it :thinking: Could you explain?

Reply:
when we have database working we can test this method, right now we cant- or have to setup all environment manually

Reply:
Is that a big effort to set up the database manually for now?

Reply:
Done